modules/features2d/include/opencv2/features2d.hpp

```diff
@@ -731,6 +773,10 @@ class CV_EXPORTS_W SimpleBlobDetector : public Feature2D
 
   CV_WRAP static Ptr<SimpleBlobDetector>
     create(const SimpleBlobDetector::Params &parameters = SimpleBlobDetector::Params());
+
+  CV_WRAP virtual void setParams(const SimpleBlobDetector::Params& params ) = 0;
```

Comment:
Do we really need `setParams()` if we already have ctor/create?
Especially "virtual".